dev/io.openliberty.org.eclipse.microprofile/openapi.3.0.bnd

```diff
@@ -18,6 +18,6 @@ Export-Package: \
   org.eclipse.microprofile.openapi.*
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;3.0}
+  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;[3.0,3.1)}
```

Comment:
This change ensures we don't automatically use any future minor version if it were to be added to oss_dependencies.maven.